src/drt/src/dr/FlexDR.cpp

```diff
@@ -104,6 +173,151 @@ int FlexDRWorker::main(frDesign* design)
   return 0;
 }
 
+inline frBlockObject* getEqObject(frBlockObject* srObj, frDesign* design)
```

Comment:
I think this function could use a bit of documentation on its purpose. I assume that this function stands for getEquivalentObject, but I'm not sure. 

I think adding the documentation and removing the acronym would help readability.

Reply:
Sure, will do.